src/util/__tests__/strip.ts

```diff
@@ -45,6 +45,34 @@ it('preserve bold with attribute', () => {
   )
 })
 
+it('empty formatting tags', () => {
+  expect(strip('<b></b>', { preserveFormatting: true })).toBe('')
+})
+
+it('empty formatting tags multiline', () => {
+  expect(
+    strip(
+      `<b>
+  
+      </b>`,
+      { preserveFormatting: true },
+    ),
+  ).toBe('')
+})
+
+it('leading and trailing whitespaces in multiline formatting tags', () => {
```

Comment:
Unit test case for leading/trailing whitespaces inside formatting tags. Apparently app used to crash when pasting such multiline tag with whitespaces.